app/scripts/modules/core/src/healthCounts/healthCounts.less

```diff
@@ -22,11 +27,6 @@ health-counts {
     float: right;
     padding-left: 5px;
   }
-  &.no-float {
```

Comment:
It's used in `AmazonLoadBalancersTag.tsx`, `TargetGroup.tsx`, `LoadBalancer.tsx`, `LoadBalancerServerGroup.tsx`, and `LoadBalancersTag.tsx`which I'm fairly confident aren't angular.